experimental/Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: all vet lint build test clean
+
+dirs=$(shell find . -maxdepth 2 -name *.go -printf "%h\n" | uniq)
+targets=$(join $(patsubst %,%/,$(dirs)),$(dirs))
+
+all: $(targets)
+
+define T
+$1/$1: $1/*.go
+	go get -tags netgo ./$$(@D)
+	go build -ldflags "-extldflags \"-static\"" -tags netgo -o $$@ ./$$(@D)
+endef
+
+$(foreach d,$(dirs),$(eval $(call T,$d)))
+
+test:
+	go test ./...
+
+clean:
+	go clean ./...
```

Comment:
Actually, can we not have a Makefile, please? Experimental stuff shouldn't be part of the regular build process.